torch/nn/modules/activation.py

```diff
@@ -868,7 +870,7 @@ class MultiheadAttention(Module):
     bias_k: Optional[torch.Tensor]
```

Comment:
Just above this line, please add this line for TorchScript support:

```python
__constants__ = ['batch_first']
```

This should be added to each of the affected modules.

Reply:
Added in `transformer.py` and `activation.py` in affected modules.